Mathlib/Order/Category/CompleteLatCat.lean

```diff
@@ -0,0 +1,102 @@
+/-
+Copyright (c) 2022 Yaël Dillies. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Yaël Dillies
+
+! This file was ported from Lean 3 source module order.category.CompleteLat
+! leanprover-community/mathlib commit e8ac6315bcfcbaf2d19a046719c3b553206dac75
+! Please do not edit these lines, except to modify the commit id
+! if you have ported upstream changes.
+-/
+import Mathlib.Order.Category.BddLatCat
+import Mathlib.Order.Hom.CompleteLattice
+
+/-!
+# The category of complete lattices
+
+This file defines `CompleteLatCat`, the category of complete lattices.
+-/
+
+set_option linter.uppercaseLean3 false
+
+universe u
+
+open CategoryTheory
+
+/-- The category of complete lattices. -/
+def CompleteLatCat :=
+  Bundled CompleteLattice
+#align CompleteLat CompleteLatCat
+
+namespace CompleteLatCat
+
+instance : CoeSort CompleteLatCat (Type _) :=
+  Bundled.coeSort
+
+instance (X : CompleteLatCat) : CompleteLattice X :=
+  X.str
+
+/-- Construct a bundled `CompleteLatCat` from a `CompleteLattice`. -/
+def of (α : Type _) [CompleteLattice α] : CompleteLatCat :=
+  Bundled.of α
+#align CompleteLat.of CompleteLatCat.of
+
+@[simp]
+theorem coe_of (α : Type _) [CompleteLattice α] : ↥(of α) = α :=
+  rfl
+#align CompleteLat.coe_of CompleteLatCat.coe_of
+
+instance : Inhabited CompleteLatCat :=
+  ⟨of PUnit⟩
+
+instance : BundledHom @CompleteLatticeHom where
+  toFun _ _ f := f.toFun
+  id := @CompleteLatticeHom.id
+  comp := @CompleteLatticeHom.comp
+  hom_ext _ _ _ _ h := FunLike.coe_injective h
+
+instance : LargeCategory.{u} CompleteLatCat :=
+  BundledHom.category CompleteLatticeHom
+
+instance : ConcreteCategory CompleteLatCat :=
+  BundledHom.concreteCategory CompleteLatticeHom
```

Comment:
please use the `by dsimp [CompleteLatCat]; infer_instance` approach instead.

Reply:
ok, did that. But this is mathport output coming from 

```
instance : large_category.{u} CompleteLat := bundled_hom.category complete_lattice_hom
instance : concrete_category CompleteLat := bundled_hom.concrete_category complete_lattice_hom
```